tailwind.config.js

```diff
@@ -266,6 +266,7 @@ module.exports = {
 			},
 			borderRadius: {
 				"2.5xl": "1.25rem",
+				table: "0.65rem",
```

Comment:
this is 10.4 pixels? does it really have to be that or can we go with 10px instead?

Reply:
Value fixed ✅